test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -1214,6 +1214,638 @@ BOOST_AUTO_TEST_CASE(slash3_slash4)
 	checkNatspec(sourceCode, "test", natspec, true);
 }
 
-BOOST_AUTO_TEST_SUITE_END()
+BOOST_AUTO_TEST_CASE(dev_default_inherit_variable)
+{
+	char const *sourceCode = R"(
+		contract C {
+			/// @notice Hello world
+			/// @dev test
+			function x() virtual external returns (uint) {
+				return 1;
+			}
+		}
+
+		contract D is C {
+			uint public override x;
+		}
+	)";
+
+	char const *natspec = R"ABCDEF({
+		"methods":
+		{
+			"x()":
```

Comment:
I think it would be easier to grasp the meaning of the test if this was just a single line